tests/python_functional/CMakeLists.txt

```diff
@@ -3,3 +3,6 @@ add_custom_target(pytest-self-check
 
 add_custom_target(pytest-check
    COMMAND ${PYTHON_EXECUTABLE} -m pytest ${PROJECT_SOURCE_DIR}/tests/python_functional/functional_tests --installdir=${CMAKE_INSTALL_PREFIX})
+
+add_custom_target(pytest-linters
+   COMMAND find ${PROJECT_SOURCE_DIR}/tests/python_functional/ -name "*.py" | xargs pre-commit run --show-diff-on-failure --config=${PROJECT_SOURCE_DIR}/tests/python_functional/.pre-commit-config.yaml --files)
```

Comment:
Unfortunately this picks the reports directory too. Can we exclude it?

Reply:
+1 sure

Reply:
fixed